modules/web-console/frontend/app/modules/agent/AgentManager.service.ts

```diff
@@ -410,7 +417,7 @@ export default class AgentManager {
         this.wsSubject
             .pipe(
                 filter((evt) => evt.requestId === requestId || evt.eventType === 'disconnected'),
-                take(1)
+                first()
```

Comment:
There is a difference between take(1) and first(): https://thecompetentdev.com/weeklyjstips/tips/4_rxjs_first_takeone/
It is safe to use first() here?

Reply:
I wanted to reduce imports and using of `first` looks more native than `take(1)`

Reply:
Well, that makes sense, so why not.